enos/enos-scenario-autopilot.hcl

```diff
@@ -290,6 +429,21 @@ scenario "autopilot" {
     value       = step.create_vault_cluster.vault_unseal_keys_b64
   }
 
+  output "vault_cluster_recovery_key_shares" {
+    description = "The Vault cluster recovery key shares"
+    value       = step.create_vault_cluster.vault_recovery_key_shares
+  }
+
+  output "vault_cluster_recovery_keys_b64" {
+    description = "The Vault cluster recovery keys b64"
+    value       = step.create_vault_cluster.vault_recovery_keys_b64
+  }
+
+  output "vault_cluster_recovery_keys_hex" {
+    description = "The Vault cluster recovery keys hex"
+    value       = step.create_vault_cluster.vault_recovery_keys_hex
+  }
+
```

Comment:
Are these required? I don't see them used elsewhere, but maybe I'm just missing it.

Reply:
That is a good question. I guess they are not used anywhere. I don't see where they are used in main as well. Anyways, I will remove them.